lib/test_summary_buildkite_plugin/input.rb

```diff
@@ -6,7 +6,7 @@
 
 module TestSummaryBuildkitePlugin
   module Input
-    WORKDIR = 'tmp/test-summary'
+    WORKDIR = "/tmp/test-summary/#{ENV['BUILDKITE_BUILD_ID']}"
```

Comment:
I'm open to keeping this as `tmp` instead of `/tmp` if that's desired for some reason. I added clean_up to be compatible for both docker and non-docker runs and make sure everything is removed